src/ecs/rendering/bundle.rs

```diff
@@ -1,87 +1,66 @@
 //! ECS rendering bundle
 
+use assets::{AssetStorage, Handle};
+use core::bundle::{ECSBundle, Result};
 use renderer::Config as DisplayConfig;
 use renderer::Rgba;
+use renderer::formats::TextureData;
 use renderer::pipe::PipelineBuild;
 use renderer::prelude::*;
-use core::bundle::{ECSBundle, Result};
 
-use assets::{AssetFuture, BoxedErr, Loader};
-use ecs::{World, DispatcherBuilder};
-use ecs::rendering::components::*;
-use ecs::rendering::resources::{AmbientColor, Factory, ScreenDimensions, WindowMessages};
+use assets::{BoxedErr, Loader};
+use ecs::{DispatcherBuilder, World};
+use ecs::rendering::resources::{AmbientColor, ScreenDimensions, WindowMessages};
 use ecs::rendering::systems::RenderSystem;
 use ecs::transform::components::*;
-
+use renderer::MaterialDefaults;
 
 /// Rendering bundle
 ///
 /// Will register all necessary components needed for rendering, along with any resources.
 /// Will also register asset contexts with the asset `Loader`, and add systems for merging
 /// `AssetFuture` into its related component.
 ///
-/// Will add `RenderSystem` as a thread local system.
-///
-/// ## Errors
-///
-/// Returns errors related to:
-///
-/// * Renderer creation
-/// * Pipeline creation
-///
-pub struct RenderBundle<P> {
-    pipe: P,
-    display_config: Option<DisplayConfig>,
-}
-
-impl<P> RenderBundle<P>
-where
-    P: PipelineBuild,
-{
-    /// Create a new render bundle with the given pipeline
-    pub fn new(pipe: P) -> Self {
-        Self {
-            pipe,
-            display_config: None,
-        }
-    }
+pub struct RenderBundle;
```

Comment:
Given this no longer adds the actual renderer I feel it's now important to include that in the documentation, and possibly provide an example as to how to add the renderer.

Reply:
Done.